src/modules/ens/ens.service.ts

```diff
@@ -194,7 +194,7 @@ export class EnsService implements OnModuleDestroy {
       domain: this._ROOT_DOMAIN,
       mode: 'ALL',
     });
-    return domains;
+    return domains.sort();
```

Comment:
Why addition of `sort` here?

Reply:
I observed less errors like `parent organization does not exists`. but maybe i am wrong, i did not count it :smile:

Reply:
Hmm, not a bad idea, maybe that makes sense. Maybe just add a comment that this is why we are sorting in this case.
```suggestion
    // Sorting to reduce "parent namespace does not exists" type error
    return domains.sort();
```

Reply:
Ok, done